### PR TITLE
Tooltips redesign

### PR DESCRIPTION
**Pull request by BeatC**
In this PR I switched Jitsi tooltips to AUI ones.

screen shot 2016-09-15 at 5 08 13 pm

### CODE REVIEW

**Comment by BeatC** [comment]
There are so much changes because I rebased the branch against `thumbnail-redesign` branch. ([Thumbnail Redesign PR](https://github.com/jitsi/jitsi-meet/pull/878)) So when that PR will be merged only changes of tooltips will be shown.

**Comment by BeatC** [comment]
fixed tests by the last commit

File: modules/UI/videolayout/SmallVideo.js [hunk 1]
```diff
@@ -217,8 +197,8 @@ SmallVideo.prototype.showAudioIndicator = function(isMuted) {
 
     if (!isMuted) {
         if (audioMutedSpan.length > 0) {
-            audioMutedSpan.popover('hide');
```
Comment (yanas) [inline modules/UI/videolayout/SmallVideo.js]: This should be handled with the new tooltip as well. If someone is muted and they hover the muted icon and click M shortcut they will unmute and the tooltip will stay.

File: modules/UI/UI.js [hunk 2]
```diff
@@ -476,7 +476,6 @@ UI.start = function () {
         $("#downloadlog").css("display", "none");
         FilmStrip.setupFilmStripOnly();
         messageHandler.enableNotifications(false);
-        $('body').popover("disable");
```
Comment (yanas) [inline modules/UI/UI.js]: This should be handled with the new tooltips as well, not just removed.

**Comment by yanas** [comment]
Tooltips have been integrated as part of a separate PR. Code from this PR has been incorporated.

**Pull request closed without merging**